sklearn/model_selection/_split.py

```diff
@@ -543,11 +543,11 @@ class StratifiedKFold(_BaseKFold):
     >>> from sklearn.model_selection import StratifiedKFold
     >>> X = np.array([[1, 2], [3, 4], [1, 2], [3, 4]])
     >>> y = np.array([0, 0, 1, 1])
-    >>> skf = StratifiedKFold(n_splits=2)
+    >>> skf = StratifiedKFold(n_splits=2, random_state=0)
```

Comment:
I don't like this. What we could do is instantiate the ``RandomState`` object only if ``shuffle=True``? That would break if someone assigned ``cv.shuffle = True`` but that seems like an odd thing to do.